hmftools-build.py

```diff
@@ -39,7 +41,76 @@ def set_version(self, version):
     @staticmethod
     def deploy_all(*modules):
         module_str = ','.join([m.name for m in modules])
-        subprocess.run(['mvn', 'deploy', '-B', '-pl', module_str, '-am', '-DdeployAtEnd=true'], check=True)
+        #subprocess.run(['mvn', 'deploy', '-B', '-pl', module_str, '-am', '-DdeployAtEnd=true'], check=True)
+        subprocess.run(['mvn', 'install', '-B', '-pl', module_str, '-am'], check=True)
+
+
+class Docker:
+    def __init__(self, module, version):
+        self.module = module
+        self.version = version
+        self.private_image = f'europe-west4-docker.pkg.dev/hmf-build/hmf-docker/{self.module}:{self.version}'
+        self.public_image = f'hartwigmedicalfoundation/{self.module}:{self.version}'
```

Comment:
private and public are terms we used to indicate one image has "private" stuff in it, while in this case its really the same image. I liked internal and external better.